readme.md

```diff
@@ -85,6 +85,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Education](https://github.com/hsavit1/Awesome-Swift-Education)
 	- [Playgrounds](https://github.com/uraimo/Awesome-Swift-Playgrounds)
 - [Python](https://github.com/vinta/awesome-python)
+	- [Python Decorator](https://github.com/lord63/awesome-python-decorator)
```

Comment:
If this is under the 'Python' list then the link can simple be called 'Decorator' or 'Decorators' even.